src/pool.c

```diff
@@ -3273,7 +3273,7 @@ miner_on_submit(json_object *message, client_t *client)
     }
 
     log_trace("Miner submitted nonce=%u, result=%s, host=%s:%u, "
-            "address=%s, rig_id=%s",
+            "address=%0.5s, rig_id=%s",
```

Comment:
`%0.5s` won't be long enough for a pool with lots of miners (you'll likely get clashes). I suggest `%0.12s`.

Reply:
This just prints the first 5 characters of the address.  How would the number of miners affect a single client's address?

Reply:
Yes, but the first char will often be the same which means you'll likely get a clash with the remaining 4 chars (when you have thousands of miners).